### ISSUE TITLE
V3.5: Max retries exceeded with url: /internal/progress

### ISSUE THREAD

**Issue by AncientMystic** [easydiffusion/easydiffusion#1853]
**Describe the bug**
V3.5 Randomly stops in the middle of generating an image and gives this error in the browser: 

`Error: HTTPConnectionPool(host='localhost', port=7860): Max retries exceeded with url: /sdapi/v1/txt2img (Caused by NewConnectionError(': Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))`

Followed by rapidly erroring out anything in queue.

**To Reproduce**
It happens at random and very often, requiring a full restart of EasyDiffusion.

**Expected behavior**
Normal image generation. 

oS & browser tried:
**Desktop (please complete the following information):**
 - OS: windows 10 22H2
 - Browser: librewolf, icecat, firefox
 - Version: 22H2

**Smartphone (please complete the following information):**
 - Device: iphone 12 
 - OS: ios 17
 - Browser: safari 
 - Version: 17.2

**Additional context**

Command line error:
```
requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=7860): Max retries exceeded with url: /internal/progress (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000000041170310>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
```
Love the v3.5 update by the way so far it is great. 👍

**Comment by cmdr2** [comment]
Hi @AncientMystic Sorry for the delay in replying to this. Yes, this is a known problem, where the Forge process crashes randomly (usually when it's under heavy load, for e.g. with Flux models). I'm investigating this problem.

**Comment by AncientMystic** [comment]
Thank you for the reply and your work on this project it is hands down the best along side comfy-ui but so much easier especially on mobile devices where comfy is basically impossible to use.

It seems to happen very easily/often especially on flux models. 

They also seem to use a ton of memory, i have 8gb vram and 96gb of ram total, I usually allocate 7-8gb vram and 48-72gb ram to the VM i run it in on proxmox, the flux models are 11-20gb files, it is amazing that somehow isn't enough ram and it maxes out and usually will only run 1-2 times if it will even run once.

**Comment by cmdr2** [comment]
Yeah, it definitely seems to be something related to Flux, but I agree with you - I don't think it's due to out-of-memory problems. Your report basically confirms that - 96 GB system RAM is more than enough to fit Flux multiple times over.

I'm able to get it to crash reliably when I alternately switch between two incompatible VAEs (i.e. SD 1.5 VAEs with Flux).

For e.g., can you please try this experiment:
1. Please download two SD 1.5 compatible VAEs to the `models/vae` folder, e.g. [vae-ft-mse-840000-ema-pruned](https://huggingface.co/stabilityai/sd-vae-ft-mse-original/resolve/main/vae-ft-mse-840000-ema-pruned.safetensors) and [color101](https://civitai.com/models/70248/color101-vae).
2. Please download and start [GPU-Z](https://www.techpowerup.com/gpuz/), a reliable GPU monitoring tool that I've used for many years.

3. Start Easy Diffusion, and select the flux model that works for you, for e.g. `flux1-dev-bnb-nf4-v2`
4. Start five tasks, with the VAE alternating between  `vae-ft-mse-840000-ema-pruned` and `color101`. E.g.
  `vae-ft-mse-840000-ema-pruned`, `color101`,  `vae-ft-mse-840000-ema-pruned`, `color101`,  `vae-ft-mse-840000-ema-pruned`.
5. Wait for the 5 tasks to finish. The first 4 tasks should fail with a normal error. The 5th task should fail with a ConnectionError.

Questions:
1. Does the 5th task die with a connection error?
2. Does the "System Memory Used" graph in GPU-Z remain mostly-flat throughout the test? For e.g. I have 32 GB system RAM, and the 5 tasks barely use the RAM, but still fail with a ConnectionError on the 5th task (ruling out the out-of-memory theory).
![image](https://github.com/user-attachments/assets/cd2c19aa-9a70-40b4-9514-248c82734e43)

Thanks

**Comment by AncientMystic** [comment]
Thank you for your reply, i will give that test a try here later when i get a minute. 👍 

From what i have noticed on my system at least, when loading flux, it will consume all available vram and memory and the error always is thrown once it consumes almost all memory and tries to grab more. 

Often this is on repeat tasks with the exact same settings as successful runs, it will end up using all 8gb vram + as much memory as i give it essentially consuming around 100gb of ram/vram in total to load a 11gb model and then crashing because it has consumed everything and cannot allocate more. 

(Although sometimes it also crashes when it clearly isnt using all the memory but either way seems weird it would consume THAT much loading the 11gb model)
